README.md

```diff
@@ -1 +1,137 @@
-# Math-magician
\ No newline at end of file
+# Math-magician
+> "Math magicians" is basically a website for everyone who loves mathematics. It is not much cause its a Single Page App that enables users to make simple calculations and read a random math-related quote. 
+
```

Comment:
- [ ] OPTIONAL
Kindly remove the file `README.md.old` from your project directory so as to keep your directory more organized.